_includes/frontpage/docs.html

```diff
@@ -1,39 +1,59 @@
 <section id="docs">
   <div class="container">
-    <h2>Documentation
-        <img class="shield" alt="GitHub package.json version"
+    <h2>Getting Started
+    <a href="https://github.com/MathJax/MathJax" class="shield">
+      <img class="shield" alt="MathJax version on GitHub"
            src="https://img.shields.io/github/package-json/v/mathjax/MathJax">
-       <img class="shield" alt="GitHub package.json version (branch)"
+      </img>
+    </a>
+    <a href="https://github.com/MathJax/MathJax/tree/legacy-v2" class="shield">
+      <img class="shield" alt="MathJax version 2 legacy version on GitHub"
            src="https://img.shields.io/github/package-json/v/mathjax/MathJax/legacy-v2">
+      </img>
+    </a>
+    <a href="https://www.npmjs.com/package/mathjax" class="shield">
+      <img class="shield" alt="MathJax npm package"
+           src="https://img.shields.io/npm/v/mathjax">
+      </img>
+    <a href="https://www.npmjs.com/package/mathjax-full" class="shield">
+      <img class="shield" alt="MathJax sources npm package"
+           src="https://img.shields.io/npm/v/mathjax-full?label=npm@sources">
+      </img>
+    </a>
+    <a href="https://www.npmjs.com/package/mathjax-node" class="shield">
+      <img class="shield" alt="MathJax version 2 legacy npm package"
+           src="https://img.shields.io/npm/v/mathjax-node?label=npm@legacy-v2">
+      </img>
```

Comment:
I would not list `mathjax-node`, at least not as `npm@legacy-v2`.  I thought this was indicating the latest v2 version on npm, as the shield doesn't make any indication that this is about `mathjax-node`.

Reply:
Removed the shield.